frontends/web/src/routes/device/bitbox02/settings.tsx

```diff
@@ -56,9 +57,6 @@ export const Settings = ({ deviceID }: TProps) => {
     route(`/passphrase/${deviceID}`);
   };
 
-  if (deviceInfo === undefined) {
-    return null;
-  }
```

Comment:
This would remove the flickering if the route would directly point to this component.

instead device settings loads bitbox02.tsx first does various checks and then eventually loads `device/bitbox02/settings.tsx` all that flickrs (even sidebar).